docs/app/Components/ComponentDoc/ComponentDescription.js

```diff
@@ -59,7 +59,8 @@ export default class ComponentDescription extends Component {
             </a>
           </code>
         </div>
-        {this.isSemanticComponent && semanticDocsLink}
+        <p>{this.props.description}</p>
+        <p>{this.isSemanticComponent && semanticDocsLink}</p>
```

Comment:
Components now show their docBlock text as their description in the doc site.  So, this:

```jsx
/**
 * It's a component
 */
export default class Component {...}
```

Would show up in the docs as:

>#### Component
>It's a component.